assayist/common/models/source.py

```diff
@@ -46,6 +46,10 @@ class SourceLocation(AssayistStructuredNode):
     component = RelationshipTo('Component', 'SOURCE_FOR', cardinality=ZeroOrOne)
     # Source locations that are embedded in this source location
     embedded_source_locations = RelationshipTo('SourceLocation', 'EMBEDS')
+    # The next version of this artifact
+    next_version = RelationshipTo('SourceLocation', 'SUPERSEDES', cardinality=ZeroOrOne)
```

Comment:
Shouldn't this be a  `PRECEDES` relationship? An older version will precede the "next" (newer) version.

Reply:
Yes, you're right. I'll flip them.

Reply:
Fixed